packages/lisk-api-client/test/api_client.js

```diff
@@ -210,6 +212,14 @@ describe('APIClient module', () => {
 					.to.have.property('headers')
 					.and.eql(customHeaders);
 			});
+
+			it('should not set User-Agent header when client options were not given', () => {
+				apiClient = new APIClient(defaultNodes, {
+					version: customHeaders.version,
```

Comment:
Actually `version`/`customHeaders.version` doesn't make sense here. Could you remove it from the other test too?